src/XenobladeChroniclesX/Mods/BladeGlobalNemesisMissionsOffline/patch_offline_nemesis.asm.bak

```diff
@@ -1,6 +1,6 @@
-[XCX_OFFLINEWE]
-moduleMatches = 0xF882D5CF, 0x30B6E091 ; 1.0.1E, 1.0.2U
-
+; not porting this backup file to JP
```

Comment:
What does this mean?

Reply:
it means that the .bak (back up file) will not be ported to JP 1.0.2 or 1.0.0